README.md

```diff
@@ -48,3 +48,8 @@ If we can hit that bullseye, the rest of the dominoes will fall like a house of
 
 As of July 2019, Java 8 is supported.
 Support for Java 6 and Java 7 has ended.
+
+## Release process
```

Comment:
not your code, but above in the readme, the method signatures have very odd formatting with the prefix comma